ads/teads.md

```diff
@@ -0,0 +1,35 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Teads
+
+## Example
+
+```html
+<amp-ad width=300 height=220
```

Comment:
The [PDF](http://mobile.teads.tv/amp/inReadbyTeads.pdf) mentions that teads have 16:9 ratio? Maybe use a `16:9` ratio for width and height for the example?